astropy/coordinates/tests/test_icrs_observed_transformations.py

```diff
@@ -14,17 +12,16 @@
 from astropy.coordinates.tests.utils import randomly_sample_sphere
 from astropy.coordinates import frame_transform_graph
 
-ra, dec, dist = randomly_sample_sphere(200)
-icrs_coords = SkyCoord(ra=ra, dec=dec, distance=dist*u.km*1e5)
 
-
-@pytest.mark.parametrize('icoo', icrs_coords)
-def test_icrs_consistency(icoo):
```

Comment:
I have no idea why the test as written iterated over all the `SkyCoord` rather than just do them in one go. Changes testing time from ~10 s to <1 s.